<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>574</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color:rgb(255, 235, 230);
background-color: rgb(255, 243, 243);*/
background-color: rgb(217, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:15px;</string>
     </property>
     <property name="text">
      <string>Benzerlik Oranını Giriniz</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 245);font-size:15px;</string>
     </property>
     <property name="plainText">
      <string>%</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:15px;</string>
     </property>
     <property name="text">
      <string>Thread Sayısını Seçiniz</string>
     </property>
    </widget>
    <widget class="QComboBox" name="threadcombobox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>72</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 245);</string>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>26</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>27</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>28</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>29</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>31</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>32</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>33</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>34</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>35</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>36</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>37</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>39</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>40</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>41</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>42</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>43</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>44</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>45</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>46</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>47</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>48</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>49</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>51</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>52</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>53</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>54</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>55</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>56</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>58</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>59</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>60</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>61</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frame3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:15px;</string>
     </property>
     <property name="text">
      <string>Benzerliği Sorgulanacak Sütunu Seçiniz</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 245);</string>
     </property>
     <item>
      <property name="text">
       <string>Product</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Issue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Company</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>State</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Complaint ID</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="Startprogram">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;font-size:18px;</string>
    </property>
    <property name="text">
     <string>Sonuçları Görmek İçin Tıklayın</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>451</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 245);</string>
     </property>
     <item>
      <property name="text">
       <string>Company</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>State</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Complaint ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ZIP Code</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:15px;</string>
     </property>
     <property name="text">
      <string>Sütunu ve Özelliği Giriniz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255,255);font-size:14px;</string>
     </property>
     <property name="text">
      <string>Ara</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 245);font-size:15px;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 245);</string>
     </property>
     <item>
      <property name="text">
       <string>Product</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Issue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Company</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>State</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Complaint ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ZIP Code</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>431</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 245);</string>
     </property>
     <item>
      <property name="text">
       <string>Product</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Issue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Company</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>State</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Complaint ID</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 245);</string>
     </property>
     <item>
      <property name="text">
       <string>Product</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Issue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Company</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>State</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Complaint ID</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);font-size:14px;</string>
     </property>
     <property name="text">
      <string>Ara</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:15px;</string>
     </property>
     <property name="text">
      <string>Aynı Olması Gereken ve Aranmak İstenen Sütunları Seçiniz</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 245);</string>
     </property>
     <item>
      <property name="text">
       <string>Product</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Issue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Company</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>State</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Complaint ID</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
